lib/govuk_app_config/govuk_healthcheck/checkup.rb

```diff
@@ -0,0 +1,46 @@
+module GovukHealthcheck
+  STATUSES = [
```

Comment:
I don't see a benefit to just returning `:ok:`, `:warning` and `:critical` from the `status` method here, since they're the same name, and we can have the `it_behaves_like "a healthcheck"` test check that they are valid like it does now.

Reply:
Well, bearing in mind I didn't write that originally so I don't know the reasons for it being there, I would say that we can't use the shared interface tests for that because they don't know enough about each class to trigger it to produce each status.

Reply:
Hmm, I suppose that's true. I don't mind too much about this, but it doesn't feel very Ruby.